### DISCUSSION TITLE
custom ldscript.ld file from sketch directory

### DISCUSSION THREAD

**Discussion by wystewart** [stm32duino/Arduino_Core_STM32#2058] [Ideas]
If possible with Arduino, it would be really nice to be able to put custom ldscript.ld file in the sketch directory.  Particularly for customized boards, that are either customized versions of the supported variants or ones own custom board that could never become a variant.

**Comment by fpistm** [comment]
Hi @wystewart 
Unfortunately, it is not easy as it is linked to Arduino boards/platform definition.
You best option is to define your own local variant as explained here:
https://github.com/stm32duino/Arduino_Core_STM32/wiki/Custom-board-based-on-a-core
